docs/samples/client/kfserving_sdk_sample.ipynb

```diff
@@ -34,6 +35,22 @@
     "from kubernetes.client import V1ResourceRequirements"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Define namespace for InferenceService deploying to. Below function defines namespace to current one if running in cluster, otherwise will deploy to `default` namespace. "
```

Comment:
"Define namespace for InferenceService deploying to. Below function defines namespace to current one if running in cluster, otherwise will deploy to `default` namespace. "

Change to 
"Define namespace where InferenceService needs to be deployed to. If not specified, below function defines namespace to the current one where SDK is running in the cluster, otherwise it will deploy to `default` namespace. "

Reply:
professional! updated. Thanks @animeshsingh